doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc

```diff
@@ -1,6 +1,6 @@
 // DO NOT EDIT - AsciiDoc file generated automatically
 
-= Heterogeneous Node Classification with HashGNN and Autotuning
+= Heterogeneous Node Classification
```

Comment:
Also here I believe name-dropping GDS concepts/algos in the title can be helpful

Reply:
```suggestion
= Heterogeneous NC: HashGNN & Autotuning
```

this is 1 char too large -- how bad is the marginal effect? Is being 1 off just as bad as 100 off? So it's either "fit within limit" or "use the best (long) name"?

Reply:
I'm not sure, I would guess there are no gradation, but... again, if it's not possible to reduce it, then let's just leave it like that!

Reply:
@Mats-SX If we can skip on the autotuning part, we can still have "Node Classification with HashGNN" and be within the limit. Otherwise, we could do "ML on nodes: HashGNN and Autotuning".

Reply:
I think it's OK to skip the autotuning part. It's not our foremost concept.